docs/computing/lumi-vs-mahti.md

```diff
@@ -0,0 +1,55 @@
+# How does LUMI-C differ from Mahti?
+
+[LUMI-C](https://docs.lumi-supercomputer.eu/computing/systems/lumic/) consists of 1536 AMD CPU nodes (2x64 cores each) with a performance of 5.6 petaflops. The node count of [Mahti](systems-mahti.md) is similar, 1404 AMD CPU nodes (2x64 cores each) corresponding to a performance of 7.5 Petaflops. Although the systems are very alike based on hardware, core count and performance, there are important differences related to access and use which are highlighted on this page.
+
+## Accessing and SSH keys
+
+To access LUMI, you need to first [create a LUMI-specific project](../accounts/how-to-create-new-project.md#how-to-create-finnish-lumi-projects). Furthermore, accessing LUMI is only possible using SSH keys, meaning that you cannot use passwords to connect through SSH like on Mahti. For instructions on how to generate an SSH key pair and uploading the public key to [MyCSC](https://my.csc.fi/), see [Setting up SSH keys](connecting.md#setting-up-ssh-keys) and the [Get started with LUMI](https://docs.lumi-supercomputer.eu/firststeps/getstarted/) pages.
+
+## Finite time projects
+
+Finnish LUMI projects have a finite duration ranging from 3 months to max. 3 years depending on the access mode:
+
+|Access mode  |Length  |Can be extended?               |
+|-------------|--------|-------------------------------|
+|Regular      |1 year  |No                             |
+|Benchmark    |3 months|No                             |
+|Extreme scale|1 year  |No                             |
+|Development  |1 year  |Yes, twice = max. 3 years total|
+
+For more details on the access modes, see the [LUMI access page on the Services for Research website](https://research.csc.fi/lumi-access). Note that LUMI users from Finland are also eligible to apply for EuroHPC Joint Undertaking (JU) resources. [See more details on European access modes here](https://www.lumi-supercomputer.eu/get-started-2021/users-in-europe/).
+
+## Software installation policy
+
+While Mahti offers frequently used applications as pre-installed modules, users are expected to compile and install the applications they intend to run on LUMI themselves. This is due to the fact that LUMI is a pan-European system with significantly more users than the national systems. Maintaining a pre-installed collection of software and licenses that would meet all users' needs on a European level would thus require excessive administration efforts. Moreover, neither the local organizations such as CSC nor the LUMI user support team (LUST) have root access to the system.
```

Comment:
I would just state that it is like this right now, and not here make so much the argument why.  The root access stuff is at least not a real reason, we do not have root on puhti and mahti either.

Reply:
and cops does have root to lumi of course